client/src/app/components/discover-import-wizard/select-platform.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Form, Text, TextContent } from "@patternfly/react-core";
+
+import { DEFAULT_SELECT_MAX_HEIGHT } from "@app/Constants";
+import { SourcePlatform } from "@app/api/models";
+import { useFetchPlatforms } from "@app/queries/platforms";
+import { toOptionLike } from "@app/utils/model-utils";
+
+import { OptionWithValue, SimpleSelect } from "../SimpleSelect";
+
+export interface SelectPlatformProps {
+  platform: SourcePlatform | null;
+  onPlatformSelected: (platform: SourcePlatform | null) => void;
+}
+
+export const SelectPlatform: React.FC<SelectPlatformProps> = ({
+  platform,
+  onPlatformSelected,
+}) => {
+  const { t } = useTranslation();
+  const { platforms } = useFetchPlatforms();
```

Comment:
are we sure the results are in cache? if not we should handle the loading state

Reply:
Never sure sure that results are in cache, but really I'm not concerned with loading state here.